### ISSUE TITLE
Riverpod example app is not working

### ISSUE THREAD

**Issue by vinothvino42** [rrousselGit/river_pod#103]
```
import 'package:flutter/material.dart';
import 'package:flutter_riverpod/flutter_riverpod.dart';

final helloWorldProvider = Provider((_) => 'Hello world');

void main() {
  runApp(
    ProviderScope(
      child: MyApp(),  // The argument type 'MyApp' can't be be assigned to the parameter type 'Widget'
    ),
  );
}

class MyApp extends ConsumerWidget {    // Classes can only extend other classes
  @override
  Widget build(BuildContext context, ScopedReader watch) {
    final String value = watch(helloWorldProvider);

    return MaterialApp(
      home: Scaffold(
        appBar: AppBar(
          title: Text('Hello Riverpod'),
        ),
        body: Center(
          child: Text(value),
        ),
      ),
    );
  }
}

```

Flutter doctor results

```
➜  riverpod_hello_world git:(master) ✗ flutter doctor -v
[✓] Flutter (Channel stable, 1.20.2, on Mac OS X 10.15.3 19D76, locale en-IN)
    • Flutter version 1.20.2 at /Users/vinothvino/Developer/flutter
    • Framework revision bbfbf1770c (11 days ago), 2020-08-13 08:33:09 -0700
    • Engine revision 9d5b21729f
    • Dart version 2.9.1

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/vinothvino/Library/Android/sdk
    • Platform android-30, build-tools 29.0.2
    • ANDROID_HOME = /Users/vinothvino/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.3

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 39.0.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.48.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.13.2

[✓] Connected device (1 available)
    • SM A505F (mobile) • RZ8M40195MN • android-arm64 • Android 10 (API 29)

• No issues found!
```

I am using the latest version of Riverpod `flutter_riverpod: ^0.7.1`. Can anyone help me? Even I tried creating a new project with newly installed Riverpod, still it's showing this exception  `Classes can only extend other classes` when I try to extend the `ConsumerWidget` class.

**Comment by rrousselGit** [comment]
Both should be fixed as part of 0.7.2, sorry for that!

**Comment by vinothvino42** [comment]
Thank you so much for the quick response.
